xcode/Subconscious/Shared/Services/UserProfileService.swift

```diff
@@ -167,26 +167,26 @@ actor UserProfileService {
     /// Because profile data is optional and we expect it will not always be present
     /// any errors are logged & handled and nil will be returned if reading fails.
     func readProfileMemo(
-        address: Slashlink
+        sphere: SphereProtocol
     ) async -> UserProfileEntry? {
+        let identity = try? await sphere.identity()
         do {
-            guard address.isProfile else {
-                throw UserProfileServiceError.attemptToReadProfileFromInvalidAddress
-            }
-            
-            let data = try? await noosphere.read(slashlink: address)
+            let data = try? await sphere.read(slashlink: Slashlink.ourProfile)
```

Comment:
Since we have a sphere reference "ourProfile" is always "this sphere's profile"

Reply:
"ourProfile" just means a peerless slashlink of `/_profile_` which should always point to _that sphere's_ profile. I can change the code to read more clearly though and use `Slashlink(slug: Slug.profile)`?